lib/widgets/HomeScreen/home_screen.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:classroom_app/widgets/CustomButton/custom_button.dart';
+import 'package:flutter/material.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.white,
+      body: Padding(
+        padding: const EdgeInsets.all(28.0),
```

Comment:
Instead of fixed paddings and sizes, use percentages.

Reply:
done